game/scripts/npc/abilities/obsidian_destroyer_arcane_orb_oaa.txt

```diff
@@ -22,7 +22,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "110 140 170 200 260 360"
+    "AbilityManaCost"                                     "110 140 170 200 230 260"
```

Comment:
After playing OD these are the changes that are needed. A buff to his early game Q manacost and a nerf to his lategame Q. OD has big mana issues early game but 0 mana issues late game (makes E really useless)

I suggest that you chang themanacost to 90/110/130/150/300/500